includes/class-newspack-popups-model.php

```diff
@@ -968,6 +969,11 @@ function( $acc, $segment_id ) {
 			];
 		}
 
+		// If the form contains registration + list info, append that to the event label.
+		if ( $has_register_form ) {
+			$event_label .= __( ' | ${formId} - ${formFields[npe]} - ${formFields[lists[]]}' );
```

Comment:
It does not seems like a good idea to send PII to GA 🤔

Reply:
Oh, good point—I wasn't thinking of the legal implications of that. 7c0d5fb removes the email from the event info.